mesonbuild/backend/backends.py

```diff
@@ -26,6 +26,12 @@
 from ..compilers import CompilerArgs, get_macos_dylib_install_name
 from collections import OrderedDict
 import shlex
+from functools import lru_cache
+
+@lru_cache(maxsize=None)
+def get_target_macos_dylib_install_name(ld):
+   return get_macos_dylib_install_name(ld.prefix, ld.name, ld.suffix, ld.soversion)
```

Comment:
__[Flake8]__

[E111] indentation is not a multiple of four


<!-- Comment by [Sider](https://sider.review) -->